ncm-download/src/main/pan/components/download/schema.pan

```diff
@@ -63,4 +63,5 @@ type download_component = extensible {
     "head_timeout" ? long
     @{seconds, total timeout for fetch of file, can be overridden per file}
     "timeout" ? long
+    "kinit_args" ? string[] with match(SELF, "[\w-.@$/]")
```

Comment:
what is the macth supposed to achieve? this only requires to have one vaild character.
if it were only these chars allowed, filenames with whitespace in the path won't be allowed etc etc

Reply:
also, can you add short description (in annotation form)